### ISSUE TITLE
Admin: Automatic light/dark theme switch

### ISSUE THREAD

**Issue by OniriCorpe** [YunoHost/issues#2556]
For now there is a manual switch to chose between light or dark theme

But an option to switch automatically according to the OS settings would be better

Tbh it should be the default behavior

**Comment by Thovi98** [comment]
You really want this feature, isn't it? ^^

https://github.com/YunoHost/issues/issues/2312

**Comment by OniriCorpe** [comment]
Ehhhh i searched if it didn’t exist lmao
I just have a bad memory
